### PR TITLE
Config scantimeout (#1)

### PR DESCRIPTION
**Pull request by dkatargin**
Add scanTimeout config parameter which use seconds for device scan timeout. This should reduce battery usage.

### CODE REVIEW

**Review by nickjturner — CHANGES_REQUESTED** [review]
Looks good, can you also add to [config.schema.json](https://github.com/nickjturner/homebridge-mi-temperature-and-humidity-sensor/blob/247a36766982137bf391745c4c0147e1bc38f74e/config.schema.json) and the [README.md](https://github.com/nickjturner/homebridge-mi-temperature-and-humidity-sensor/blob/main/README.md)

**Comment by dkatargin** [comment]
Thanks for reviewing! I added the requested changes to this commit: 6890d36c7382aedf4180508e60e2d184ed9ffd2c

File: README.md [hunk 1]
```diff
@@ -45,14 +45,16 @@ For the BLE connection look at the noble [prerequisites](https://github.com/aban
       {
           "accessory": "MiSensor",
           "name": "Living Room Mi Sensor",
-          "address": "a4:c1:38:b9:3f:72"
+          "address": "a4:c1:38:b9:3f:72",
+          "scanTimeout": 3600,
```
Comment (nickjturner) [inline README.md]:
```suggestion
          "scanTimeout": 3600
```

**Review by nickjturner — APPROVED** [review]
LGTM

**Pull request merged**